custom_components/ha_washdata/manager.py

```
@@ -234,6 +258,20 @@ def _pn_create(
 
+def _pn_dismiss(hass: HomeAssistant, notification_id: str) -> None:
+    """Best-effort persistent notification dismissal."""
+    try:
+        components = getattr(cast(Any, hass), "components", None)
+        pn = getattr(cast(Any, components), "persistent_notification", None)
+        if pn is None:
+            return
+        result = pn.async_dismiss(notification_id)
+        if inspect.iscoroutine(result):
+            hass.async_create_task(result)
+    except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)